src/System.Runtime.Extensions/tests/System/IO/PathTests_Windows.netfx.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
+    [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]
+    public class PathTests_NetFX : PathTestsBase
```

Comment:
Nit: File name vs class name mismatch

Reply:
This one is a bit awkward. I really want to keep it in the build as it makes it much easier to work with, but I don't want to litter the file with [SkipOnTargetFramework] and yet still clearly show the relationship when browsing the solution explorer. :/